libraries/webFrontend.js

```diff
@@ -154,10 +162,18 @@ exports.printFolder = function (objects, objectsPath, debug, objectInterfaceName
         }
 
         // world objects are always initialized true regardless of target data
-        if (newObject[thisObjectKey].isWorldObject) {
-            newObject[thisObjectKey].initialized = true;
-        }
 
+            if (newObject[thisObjectKey].isWorldObject) {
```

Comment:
Empty if statement

Reply:
why is it empty?

Reply:
There is no code contained in this if statement so it may as well be deleted or commented out

Reply:
commented